ncolony/client/tests/test_heart.py

```diff
@@ -41,6 +41,20 @@ def _cleanup():
     case.addCleanup(_cleanup)
     os.environ = myEnv
 
+def _getSelf(method):
+    if hasattr(method, 'im_self'):
```

Comment:
[`hasattr` is broken on Python 2](https://hynek.me/articles/hasattr/).  Use `getattr `instead:

```python
_missing = missing
self = getattr(method, 'im_self', _missing)
```